### PR TITLE
Depsrems

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

File: site/en/blog/deps-rems-105/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## Remove WebSQL in non-secure contexts
+
+WebSQL in non-secure contexts is deprecated. The Web SQL Database standard was first proposed in April 2009 and abandoned in November 2010. Gecko never implemented this feature and WebKit deprecated it in 2019. The W3C encourages [Web Storage](https://developer.mozilla.org//docs/Web/API/Web_Storage_API) and [IndexedDb](https://developer.mozilla.org//docs/Web/API/IndexedDB_API) for those needing alternatives.
+
+Developers should expect that WebSQL itself will be deprecated and removed when usage is low enough. For more information, see [Deprecating Web SQL](/blog/deprecating-web-sql).
+
+## CSS default keyword is disallowed in custom identifiers
+
+The CSS keyword ['default' is no longer allowed](https://chromestatus.com/feature/5096490737860608) within CSS custom identifiers, which are used for many types of user-defined names in CSS (for example, names created by @keyframes rules, counters, @container names, custom layout or paint names). This adds `'default'` to the list of names that are restricted from use in custom identifiers, specifically `'inherit'`, `'initial'`, `'unset'`, `'revert'`, and `'revert-layer'`.
```
Comment (rachelandrew) [inline site/en/blog/deps-rems-105/index.md:28]:
```suggestion
The CSS keyword ['default' is no longer allowed](https://chromestatus.com/feature/5096490737860608) within CSS custom identifiers, which are used for many types of user-defined names in CSS (for example, names created by `@keyframes` rules, counters, `@container` names, custom layout or paint names). This adds `'default'` to the list of names that are restricted from use in custom identifiers, specifically `'inherit'`, `'initial'`, `'unset'`, `'revert'`, and `'revert-layer'`.
```

File: site/en/blog/deps-rems-105/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## Deprecations in the Navigation API
+
+The `transitionWhile()` and `restoreScroll()` methods are also deprecated in this release, and we expect to remove them in 108. Developers who need this functionality should use the new `intercept()` and `scroll()` methods. For explanations of the problems with the existing methods and examples of using the new, see [link here].
```
Comment (rachelandrew) [inline site/en/blog/deps-rems-105/index.md:32]: Missing link

File: site/en/blog/deps-rems-105/index.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## Deprecate non-ASCII characters in cookie domain attributes
+
+To align with the latest spec ([RFC 6265bis](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis/#section-5.5)), [Chromium will soon reject](https://www.chromestatus.com/feature/5534966262792192) cookies with a "Domain" attribute that contains a non-ASCII character (for example, Domain=éxample.com).\
```
Comment (rachelandrew) [inline site/en/blog/deps-rems-105/index.md:36]:
```suggestion
To align with the latest spec ([RFC 6265bis](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis/#section-5.5)), [Chromium will soon reject](https://www.chromestatus.com/feature/5534966262792192) cookies with a "Domain" attribute that contains a non-ASCII character (for example, Domain=éxample.com).
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]
There's a missing link in this, you may know but thought I'd just flag it up.

File: site/en/blog/deps-rems-105/index.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+Developers should expect that WebSQL itself will be deprecated and removed when usage is low enough. For more information, see [Deprecating Web SQL](/blog/deprecating-web-sql).
+
+## CSS default keyword is disallowed in custom identifiers
+
+The CSS keyword ['default' is no longer allowed](https://chromestatus.com/feature/5096490737860608) within CSS custom identifiers, which are used for many types of user-defined names in CSS (for example, names created by `@keyframes` rules, counters, `@container` names, custom layout or paint names). This adds `'default'` to the list of names that are restricted from use in custom identifiers, specifically `'inherit'`, `'initial'`, `'unset'`, `'revert'`, and `'revert-layer'`.
+
+## Deprecations in the Navigation API
+
+The `transitionWhile()` and `restoreScroll()` methods are also deprecated in this release, and we expect to remove them in 108. Developers who need this functionality should use the new `intercept()` and `scroll()` methods. For explanations of the problems with the existing methods and examples of using the new, see [link here].
```
Comment (jpmedley) [inline site/en/blog/deps-rems-105/index.md:32]:
```suggestion
The `transitionWhile()` and `restoreScroll()` methods are also deprecated in this release, and we expect to remove them in 108. Developers who need this functionality should use the new `intercept()` and `scroll()` methods. For explanations of the problems with the existing methods and examples of using the new, see [Changes to NavigateEvent](https://developer.chrome.com/blog/navigateevent-intercept/).
```

**Comment by jpmedley** [comment]
Issues addressed. PTAL

**Review by rachelandrew — APPROVED** [review]

File: site/en/blog/deps-rems-105/index.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+Developers should expect that WebSQL itself will be deprecated and removed when usage is low enough. For more information, see [Deprecating Web SQL](/blog/deprecating-web-sql).
+
+## CSS default keyword is disallowed in custom identifiers
+
+The CSS keyword ['default' is no longer allowed](https://chromestatus.com/feature/5096490737860608) within CSS custom identifiers, which are used for many types of user-defined names in CSS (for example, names created by `@keyframes` rules, counters, `@container` names, custom layout or paint names). This adds `'default'` to the list of names that are restricted from use in custom identifiers, specifically `'inherit'`, `'initial'`, `'unset'`, `'revert'`, and `'revert-layer'`.
+
+## Deprecations in the Navigation API
+
+The `transitionWhile()` and `restoreScroll()` methods are also deprecated in this release, and we expect to remove them in 108. Developers who need this functionality should use the new `intercept()` and `scroll()` methods. For explanations of the problems with the existing methods and examples of using the new, see [Changes to NavigateEvent](https://developer.chrome.com/blog/navigateevent-intercept/).
```
Comment (rachelandrew) [inline site/en/blog/deps-rems-105/index.md:32]:
```suggestion
The `transitionWhile()` and `restoreScroll()` methods are also deprecated in this release, and we expect to remove them in 108. Developers who need this functionality should use the new `intercept()` and `scroll()` methods. For explanations of the problems with the existing methods and examples of using the new, see [Changes to NavigateEvent](/blog/navigateevent-intercept/).
```

**Pull request merged**